microservices.yml

```diff
@@ -77,30 +98,35 @@ commands:
           command: |
             eval $(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)
             docker push $TAG_COMMIT
-            docker push $TAG_CI
+            docker push $TAG_BRANCH
             docker push $TAG_LATEST
+            [ ! -z $TAG_GITTAG ] && docker push $TAG_GITTAG || true
 
-  install-npm-modules-with-cache:
-    description: Installs npm modules from circleci cache (if it exists) and saves results back to cache
+  run-unit-tests-in-docker:
+    description: Runs tests in docker container
     steps:
-      - restore_cache:
-          key: rooms-npm-cache-{{ checksum "package.json" }}
       - run:
-          name: Install NPM dependencies
+          name: Run unit tests in docker container
           command: |
-            npm install
-      - save_cache:
-          key: rooms-npm-cache-{{ checksum "package.json" }}
-          paths:
-            - ./node_modules
-
-  run-docker-compose-tests:
-    description: Runs docker compose up tests with proper exit code verification to fail build
+            docker run --env LOG_TO_CONSOLE=true --env LOG_TO_FILE=true --env LOGGING_LEVEL=info --env-file common.env --env-file tables.env --env-file assa.env --env-file dorma.env -it ${TAG_COMMIT} npm run test:unit
+  run-integration-tests-in-docker:
+    description: Runs integration tests in docker container
     steps:
       - run:
-          name: Run docker compose up test with exit code flag
+          name: Run integration tests in docker container
           command: |
-            docker-compose up --exit-code-from test test
+            docker run --env LOG_TO_CONSOLE=true --env LOG_TO_FILE=true --env LOGGING_LEVEL=info --env-file common.env --env-file tables.env --env-file assa.env --env-file dorma.env -it ${TAG_COMMIT} npm run test:integration
+  run-lint-in-docker:
+    description: Runs lint in docker container
+    steps:
+      - run:
+          name: Run lint in docker container
+          command: |
+            docker run -v /home/circleci/project/reports:/usr/src/app/reports -it ${TAG_COMMIT} npm run ci:lint
```

Comment:
@misiek-sf Does `npm run ci:lint` script exist in all repos yet?

Reply:
yep (gotten there with `linting and stuff`)